Minor changes

fellipec opened the pull request:
Minor changes before I start to work in the main branch.

xeger commented on mission_kss.ks:
```diff
@@ -0,0 +1,40 @@
+/////////////////////////////////////////////////////////////////////////////
+// Idempotent mission script: Voyage to Mun.
```
Copy/paste error (comment still describes mun transfer, not KSS rendezous).

xeger commented on node.ks:
```diff
@@ -18,21 +20,24 @@ global nodeStageFuelInit is 0.
+// If have time, wait to ship almost align with maneuver node.
+// If have little time, wait at least to ship face inside 40º cone from the node. This prevents backwards burns.
+wait until (vdot(facing:forevector, nodeFacing:forevector) >= 0.995) or
+           ((nodeNd:eta <= nodeDob / 2) and (vdot(facing:forevector, nodeFacing:forevector) >= 0.94)). WAIT 5.
```
Could the hardcoded `WAIT 5` ever cause us to wait too long and miss the beginning of our burn window?

fellipec commented:
Could the hardcoded WAIT 5 ever cause us to wait too long and miss the beginning of our burn window?
True. Added that while I was testing and forgot to remove. I’ll delete in next commit.
Anyway if the node is too soon and ship is turned completely away from its direction, it will miss the begining of burn window. Before it even start to burn to the wrong direction, I added the code just before the wait 5 so the script at least wait the ship to turn facing more or less in the direction of the burn. I think is “less worse” than begin burning in the wrong direction.

Enviado do Email<https://go.microsoft.com/fwlink/?LinkId=550986> para Windows 10

________________________________
De: Tony Spataro <notifications@github.com>
Enviado: Monday, November 6, 2017 4:25:34 PM
Para: xeger/kos-ramp
Cc: Luiz Fellipe Carneiro; State change
Assunto: Re: [xeger/kos-ramp] Minor changes (#9)


@xeger commented on this pull request.

________________________________

In node.ks<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fxeger%2Fkos-ramp%2Fpull%2F9%23discussion_r149161347&data=02%7C01%7Cfellipec%40outlook.com%7C920ddc9d8a8341886e7508d52543c58a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636455895353465752&sdata=4JwW%2Bu%2Bw2Vfs45ZNHy7kvXSsweLmr107ycyKDMGP4l0%3D&reserved=0>:

>
 // estimate burn direction & duration
 global nodeAccel is uiAssertAccel("Node").
 global nodeFacing is lookdirup(nodeNd:deltav, ship:facing:topvector).
 global nodeDob is (nodeNd:deltav:mag / nodeAccel).

 uiDebug("Orient to burn").
-wait until vdot(facing:forevector, nodeFacing:forevector) >= 0.995 or nodeNd:eta <= nodeDob / 2.
+// If have time, wait to ship almost align with maneuver node.
+// If have little time, wait at least to ship face inside 40º cone from the node. This prevents backwards burns.
+wait until (vdot(facing:forevector, nodeFacing:forevector) >= 0.995) or
+           ((nodeNd:eta <= nodeDob / 2) and (vdot(facing:forevector, nodeFacing:forevector) >= 0.94)). WAIT 5.


Could the hardcoded WAIT 5 ever cause us to wait too long and miss the beginning of our burn window?

—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fxeger%2Fkos-ramp%2Fpull%2F9%23pullrequestreview-74512756&data=02%7C01%7Cfellipec%40outlook.com%7C920ddc9d8a8341886e7508d52543c58a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636455895353465752&sdata=Njjl1nBHtWPMBmnQeH7Db%2Figs7GpIIJmQDSKs8sWPHU%3D&reserved=0>, or mute the thread<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAQAlIXmINDwhI2o8RNu58U-g4aKbD7m7ks5sz08egaJpZM4QTbNd&data=02%7C01%7Cfellipec%40outlook.com%7C920ddc9d8a8341886e7508d52543c58a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636455895353465752&sdata=EDCs5AKEQ31y1Tx9TP6POyQIToqRs%2BLV3Y3G%2FT8MPbQ%3D&reserved=0>.
